### ISSUE TITLE
Pickers:[Keros bug]: ARIA role='list' must contain one or more list items

### ISSUE THREAD

**Issue by saini-ankish** [OfficeDev/office-ui-fabric-react#8453]
**Environment Information**

- **Package version(s)**: >6.101.0
- **Browser and OS versions**: chrome version 69.0.3497.100 and OS version windows 10

- I saw similar bugs logged earlier, but issue is still there.

Bug repro URL: https://developer.microsoft.com/en-us/fabric#/components/pickers

![Picker_Keros bug](https://user-images.githubusercontent.com/44394750/54903430-e3632d00-4f01-11e9-84a9-787cec44b7db.PNG)

**Actual behavior**:
**Keros issue** : Certain ARIA roles must contain particular children

**Expected behavior**:
keros should not give any issue

**Comment by Vitalius1** [comment]
I've investigated a bit this issue and it seems that it's caused by [this WCAG](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/List_role#Description) rule `role="list"` set on this element:
https://github.com/OfficeDev/office-ui-fabric-react/blob/ae66c05a20ddf23c37fd92a1d3528815e15cc45c/packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx#L236-L238
The issue it appears to be that the list needs to have one or more listitems but sisnce the picker has no picked items yet, we have it flagged by Keros. One solution would be to have a conditional checking if there is no items selected then set `role` to `presentation` and to `list` if we have already selected some. @joschect would love to hear your opinion on this as this gets flagged in PeoplePicker too.
